<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAddRasterLayer</class>
 <widget class="QDialog" name="DlgAddRasterLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Postgis Raster layer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Postgis' connection:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available tables</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Schema</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Table</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SRID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pixel Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Is external file</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Is regular block</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pixel size x </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pixel size y</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Table name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>SQL Query:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Query Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget_2"/>
   </item>
   <item row="14" column="0">
    <widget class="QCheckBox" name="selAllCheckBox">
     <property name="text">
      <string>select All</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="contLinesCheckBox">
     <property name="text">
      <string>get countour lines too</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>RUN SQL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgAddRasterLayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgAddRasterLayer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
